set.mm

```diff
@@ -25753,42 +25801,68 @@ equal class abstractions (inference form).  (Contributed by NM,
     abbidOLD $p |- ( ph -> { x | ps } = { x | ch } ) $=
       ( wb wal cab wceq alrimi abbi sylib ) ABCGZDHBDICDIJANDEFKBCDLM $.
 
-    $d ps y $.  $d ch y $.  $d x y $.
+    $d ph y $.  $d ps y $.  $d ch y $.  $d x y $.
     $( Equivalent wff's yield equal class abstractions (deduction form).
        (Contributed by NM, 21-Jun-1993.)  (Revised by Mario Carneiro,
-       7-Oct-2016.)  Avoid ~ ax-11 .  (Revised by Wolf Lammen, 4-May-2023.) $)
+       7-Oct-2016.)  Avoid ~ ax-11 and ~ ax-10 .  (Revised by Wolf Lammen,
+       6-May-2023.) $)
     abbid $p |- ( ph -> { x | ps } = { x | ch } ) $=
-      ( vy wb wal cab wceq alrimi wsb wcel stdpc4v sbbiv sylib df-clab 3bitr4g
-      cv eqrdv syl ) ABCHZDIZBDJZCDJZKAUCDEFLUDGUEUFUDBDGMZCDGMZGTZUENUIUFNUDUC
-      DGMUGUHHUCDGOBCDGPQBGDRCGDRSUAUB $.
+      ( vy sbbid abbilem ) ABCDGABCDGEFHI $.
   $}
 
   ${
-    $d x ph $.
+    $d x y ph $.  $d y ps $.  $d y ch $.
     abbidv.1 $e |- ( ph -> ( ps <-> ch ) ) $.
     $( Equivalent wff's yield equal class abstractions (deduction form).
-       (Contributed by NM, 10-Aug-1993.) $)
+       (Contributed by NM, 10-Aug-1993.)  Avoid ~ ax-12 , based on an idea of
+       Steven Nguyen.  (Revised by Wolf Lammen, 6-May-2023.) $)
```

Comment:
```suggestion
       (Contributed by NM, 10-Aug-1993.)  Avoid ~ ax-10 and ~ ax-12 , based on
       an idea of Steven Nguyen.  (Revised by Wolf Lammen, 6-May-2023.) $)
```

based on abbii, it seems ~ ax-10 is also saved

Reply:
All three ax-10, ax-11 and ax-12 are finally saved, but two of them are inherited from abbid, that was rewritten, too.  The modification to abbidv addresses the final ax-12 only.  I do not list cascading effects explicitly.  Otherwise I would have to update many more comments, not only in the theorems listed here.

I put your name here to let your contribution appear less trivial than the the proof of abbii now expresses.  An astute reader following the dates in the tags will see that your modification was first, and so the proof idea was moved around.

Let me know if my addition is not sufficient.